browser/js/pipeline/pipeline.stage/pipeline.stage.candidate/panelTemplates/kit.html

```diff
@@ -1,3 +1,18 @@
-<div class="kit-panel">
-{{data}}
-</div>
\ No newline at end of file
+<div class="kit-panel" ng-controller="kitPanelCtrl" >
+    <div class="question-field" ng-repeat="question in data.questions">
+    	<!-- @Omri passing in 
```

Comment:
Nope. Not quite sure I understand the problem question. I'll reach out IRL so we can chat about it.